docs/portal/transports/delete-transport.md

```diff
@@ -1,7 +1,7 @@
 # Delete a Transport
 
-To delete a transport, search the transport you want to delete in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Delete".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
```

Comment:
Please add the word "the" as follows: "... from the left pane of the HDT portal."